druid-shell/src/backend/mac/window.rs

```diff
@@ -1198,7 +1214,21 @@ impl WindowHandle {
     pub fn show_titlebar(&self, _show_titlebar: bool) {}
 
     // Need to translate mac y coords, as they start from bottom left
-    pub fn set_position(&self, position: Point) {
+    pub fn set_position(&self, mut position: Point) {
+        // TODO: Maybe collin can get this into a state where modal windows follow the parent?
```

Comment:
```suggestion
        // TODO: Maybe @cmyr can get this into a state where modal windows follow the parent?
```

(@colin would also be fine, but this should be easier for people to understand)